static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -730,6 +730,55 @@
 
 }
 
+// Fabric
+// =============================================================================
+
+// Layout
+
+.ad-slot--fabric {
+
+    &.ad-slot--top-banner-ad {
+        width: auto;
+        padding-bottom: 0;
+        padding-left: 0;
+    }
+
+    @include mq(mobile, tablet) {
+        // Full width on mobile fronts
+        width: auto;
+    }
+}
+
+.fabric {
+    height: 250px;
+    transition: height .8s;
+
+    // This is just for demonstration purposes, whilst the ad is populated with ipsum text
```

Comment:
To be honest I don't like how the white & blue font looks like on that colourful background. Is this a version that we are going to demonstrate during the ad week?